pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -226,6 +247,17 @@ func EnsureProjectConfigured(r *ReferenceAdapter) (gcputil.OperationResult, erro
 		return gcputil.RequeueWithError(operrors.Wrap(err, "error creating credentials"))
 	}
 
+	if r.isCCS() {
+		r.logger.V(1).Info("Configuring Service Account Permissions for Console Access")
```

Comment:
Is this message correct?  Is the operator configuring the service account?  Or is it configuring IAM to allow console access?

Reply:
`Configuring IAM to allow console access` is better! I will update it.